setup.py

```diff
@@ -3,7 +3,7 @@
 from setuptools import setup
 
 setup(
-    name="taurusgui-linacsynoptic",
+    name= "taurusgui-linacsynoptic",
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/setup.py
+++ b/setup.py
@@ -3,7 +3,7 @@
 from setuptools import setup
 
 setup(
-    name= "taurusgui-linacsynoptic",
+    name="taurusgui-linacsynoptic",
     version="1.4.0",
     description="Synoptic for the MAX IV linear accellerator.",
     author="Johan Forsberg",
```